tests/store/test_store_api.py

```diff
@@ -38,6 +38,24 @@ def test_analysis(analysis_store: MockStore):
     assert analysis_obj is None
 
 
+def test_update_analysis_from_slurm_run_status(analysis_store: MockStore, squeue_stream_jobs: str):
+    """Test updating analysis jobs when given squeue results."""
+    # GIVEN an analysis and a squeue stream
+    analysis: Analysis = analysis_store.get_query(table=Analysis).first()
+    assert not analysis.failed_jobs
+
+    # WHEN updating the analysis
+    analysis_store.update_analysis_from_slurm_run_status(analysis_id=analysis.id)
+    updated_analysis: Analysis = analysis_store.get_analysis(
+        case_id=analysis.family,
+        started_at=analysis.started_at,
+        status=TrailblazerStatus.RUNNING,
+    )
+
+    # THEN it should update the analysis jobs
+    assert updated_analysis.failed_jobs
```

Comment:
same comment as above update analysis with RUNNING and looking for failed_jobs

Reply:
Yes, it is misnamed in the code. It actually contains all jobs. I will make an issue so we can fix it in a separate PR.